packages/core/src/controllers/pairing.ts

```diff
@@ -187,7 +187,7 @@ export class Pairing implements IPairing {
 
   public activate: IPairing["activate"] = async ({ topic }) => {
     this.isInitialized();
-    const expiry = calcExpiry(THIRTY_DAYS);
+    const expiry = calcExpiry(ONE_MINUTE);
```

Comment:
shouldn't be 5 minutes? also you don't need an activate method anymore
look into the doc https://www.notion.so/walletconnect/Improving-the-Sign-SDK-Pairing-Expiry-Reduction-d4cf787ecd9f4169a6887d45bb5d5069

```Deprecation: The concepts of active and inactive pairings will no longer apply. Pairings will only exist until the first session is created. The pairing_delete, pairing_extend, and pairing_ping methods will be deprecated.```